### PR TITLE
qa: Various make check fixes

### PR DESCRIPTION
**Pull request by sebastian-philipp**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/CMakeLists.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ endif()
 
 if(WITH_TESTS)
   include(AddCephTest)
-  add_tox_test(qa TOX_ENVS py3 flake8 import-tasks)
+  add_tox_test(qa TOX_ENVS py3 flake8 import-tasks mypy)
```
Comment (tchaikov) [inline qa/CMakeLists.txt:8]:
```
mypy run-test: commands[0] | mypy .
workunits/fs/misc/filelock_interrupt.py:26: error: Module has no attribute "F_OFD_GETLK"; maybe "F_GETLK"?
workunits/fs/misc/filelock_interrupt.py:27: error: Module has no attribute "F_OFD_SETLK"; maybe "F_SETLK"?
workunits/fs/misc/filelock_interrupt.py:28: error: Module has no attribute "F_OFD_SETLKW"; maybe "F_SETLKW"?
Found 3 errors in 1 file (checked 232 source files)
```

probably could allow following patch
```diff
diff --git a/qa/workunits/fs/misc/filelock_interrupt.py b/qa/workunits/fs/misc/filelock_interrupt.py
index 5ebc480ba8e..b261d74fbc0 100755
--- a/qa/workunits/fs/misc/filelock_interrupt.py
+++ b/qa/workunits/fs/misc/filelock_interrupt.py
@@ -23,9 +23,9 @@ def timeout(seconds):
 """
 introduced by Linux 3.15
 """
-fcntl.F_OFD_GETLK = 36
-fcntl.F_OFD_SETLK = 37
-fcntl.F_OFD_SETLKW = 38
+setattr(fcntl, "F_OFD_GETLK", 36)
+setattr(fcntl, "F_OFD_SETLK", 37)
+setattr(fcntl, "F_OFD_SETLKW", 38)
 
 
 def main():
```

Comment (sebastian-philipp) [reply qa/CMakeLists.txt:8]: Fascinating. I did run `tox -e mypy` locally without seeing this error.

Comment (sebastian-philipp) [reply qa/CMakeLists.txt]: fixed.

**Comment by sebastian-philipp** [comment]
changelog: 

* qa: convert test_import.py to pytest

File: qa/tasks/tests/test_cephadm.py [hunk 2]
```diff
@@ -1,70 +0,0 @@
-from tasks import cephadm
```
Comment (tchaikov) [inline qa/tasks/tests/test_cephadm.py:1]: @sebastian-philipp i don't understand how this is related to the renaming. could you elaborate it a little bit?

**Review by tchaikov — COMMENTED** [review]
apart from the removal of `qa/tasks/tests/test_cephadm.py`, LGTM.

File: qa/tasks/tests/test_cephadm.py (hunk 2 above)
Comment (sebastian-philipp) [reply qa/tasks/tests/test_cephadm.py:1]: should be in a different commit, indeed. Thanks

**Comment by sebastian-philipp** [comment]
> apart from the removal of `qa/tasks/tests/test_cephadm.py`, LGTM.

pushed

**Review by tchaikov — APPROVED** [review]

**Pull request merged**